CONTRIBUTING.md

```diff
@@ -1,45 +1,94 @@
+regarding the feature you would like to see in SunPy.
 
-* Get a [GitHub Account](https://github.com/signup/free)
-* [Fork](https://help.github.com/articles/fork-a-repo) SunPy
-* Make your changes and push to your fork.
-* Create a [Pull Request](https://help.github.com/articles/using-pull-requests).
 
-For more detailed information on contributing code see our 
-[Developers Guide](http://docs.sunpy.org/en/stable/dev.html).
+## Finding Things to Contribute to
+
+If you want to contribute to SunPy but don't know where to start, we maintain a
+list of issues that are good as a starting place, these can be found under the
+[package-novice](https://github.com/sunpy/sunpy/issues?q=is%3Aissue+is%3Aopen+label%3Apackage-novice)
+label.
+
+
+## Making a Code or Documentation Contribution
+
+To make a contribution to the SunPy code or documentation you will need to use
+git and GitHub, which are the tools SunPy uses to manage changes to the code and
+documentation.
+
+1. Get a [GitHub Account](https://github.com/signup/free)
+1. [Fork](https://help.github.com/articles/fork-a-repo) SunPy on GitHub
+1. Install the developer version of SunPy using git. 
+1. Make your changes and push to your fork.
+1. Create a [Pull Request](https://help.github.com/articles/using-pull-requests).
+
+If any of the above steps are alien to you check out our
+[Developers Guide](http://docs.sunpy.org/en/latest/dev.html). (Suggesting
+improvements to the developers guide is an excellent way to contribute!!)
+
 
 ## Pull Request Review
 
 When you submit a pull request it can be reviewed and commented on by anyone.
+Everyone's contributions are reviewed, and no-one's contributions are accepted
+in to SunPy without being reviewed. We do this to try to maintain a consistent
+coding style and high quality code.
+
 For the code to be accepted into SunPy it must meet the following criteria:
 
 * Conform to code quality standards, primarily [PEP8](http://legacy.python.org/dev/peps/pep-0008/).
-* Have close to 100% unit test coverage.
+* Have close to 100% unit test coverage (see [testing](http://docs.sunpy.org/en/latest/dev.html#testing)).
 * Have API documentation generated by astropy's automodapi extenstion where appropriate.
-* Contain an entry in the CHANGELOG.md file.
+* Contain an entry in the [CHANGELOG.md](https://github.com/sunpy/sunpy/blob/master/CHANGELOG.md) file.
 * Be approved by at least two SunPy contributors.
 
 
 ## Pull Requests to non-master Branches
 
 From time to time, large feature development work may occur in branches other than master, and all 
 maintained releases currently have a branch i.e. `0.6`.
-When making a PR to one of these branches, please put the name of the branch in square brackets at 
+When making a PR to one of these branches (not master), please put the name of the branch in square brackets at 
 the beginning of the PR, i.e. `[0.6] My bug fix`. This makes it easier to filter and review these PRs.
```

Comment:
the beginning of the PR, i.e. `[0.6] My bug fix`. This makes it easier to filter and review these PRs on GitHub.